.rhcicd/clowdapp-aggregator.yaml

```diff
@@ -80,7 +80,7 @@ parameters:
   required: true
 - name: PROMETHEUS_PUSHGATEWAY
   description: Prometheus Pushgateway URL
-  value: ""
+  value: changeme
```

Comment:
```suggestion
  value: CHANGE_ME
```

Reply:
Oups, was too fast, will change if that worked

Reply:
Don't worry about it, it's not that important.